napari/_qt/qt_dims.py

```diff
@@ -328,3 +328,40 @@ def focus_down(self):
         else:
             index = (displayed.index(self.last_used) - 1) % len(displayed)
             self.last_used = displayed[index]
+
+    def play_dim(self, axis: int = 0, fps: float = 10):
```

Comment:
how do we feel about allowing multiple axes to be playing at the same time? it seems pretty strange to me, in which case we should probably add a `self.stop()` to stop any existing playing inside this function

Reply:
yeah, I agree it should be only a single axis at a time (at least until some real need otherwise has been demonstrated).  The current code already accomplishes that though, since there is only a single `self.play_timer`... so if play gets called again on another axis, the older `QTimer` gets garbage collected and playback on that axis stops.  Do you think this should be made more explicit?

Reply:
ah ok nice I missed that - yeah, if you don't mind, I think including the explicit call to `stop` and a comment saying only one axis can be playing at a time makes it more clear what's going on